apps/server/src/stores/token-player-map.ts

```diff
@@ -1,19 +1,35 @@
 import { AuthToken, PlayerId, socketId } from '@razor/models';
 
 interface MapData {
-  playerId: PlayerId;
+  playerId?: PlayerId;
   socketId: socketId;
 }
 
+interface mapAsObject {
```

Comment:
Capitalize the first letter

Reply:
fixed 817c1a8bd87dc25122f7236fa14e89c190b9dbb7 in PR #81